spec/awesome_spawn_spec.rb

```diff
@@ -15,6 +15,51 @@
       subject.send(run_method, "true", :params => {:user => "bob"}, :env => {"VAR" => "x"})
     end
 
+    it "supports pipes with :params" do
+      cmds = [
+        "true",
+        ["true", :params => { :arg1 => "one", :"no-arg2" => nil }]
```

Comment:
Minor, but that `:"no-arg2"` can just be `:no_arg2`

Reply:
Ah, that is right.  I will make the change.

Reply:
"Done goof'd" with a typo on the first pass at fixing this, but this is all fixed now.